ports/servoshell/desktop/dialog.rs

```diff
@@ -54,6 +62,12 @@ impl Dialog {
         Dialog::File(dialog)
     }
 
+    pub fn new_alert_dialog(message: String, sender: IpcSender<()>) -> Self {
+        let alert_dialog = AlertDialog { message, sender };
+
+        Dialog::Alert(alert_dialog)
```

Comment:
I think this is a little nicer as a single line:

```suggestion
        Dialog::Alert(AlertDialog { message, sender })
```